.config.json

```diff
@@ -1,8 +1,10 @@
 {
-  "db":{
-    "url":"mongodb://localhost",
-    "port":"27017",
-    "databaseName": "heartbeat",
-    "collection": "heartbeat"
+  "express": {
+    "database":{
```

Comment:
spaaaaaaaaaaaaaaaaaaaaace

Reply:
Sorry lint is not ready i.e. the google style is missing in config